src/auxiliary/mpi_arrays.jl

```diff
@@ -216,13 +221,15 @@ Instead of checking something like `any(isnan, u)`, this function just checks
 global communication is required and all ranks will return the same result.
 
 You should pass this function as keyword argument
-`unstable_check=Trixi.ode_unstable_check`
+`unstable_check=ode_unstable_check`
 of `solve` when using error-based step size control with MPI parallel execution
 of Trixi.jl.
+
+See [`https://diffeq.sciml.ai/stable/basics/common_solver_opts/#Miscellaneous`](https://diffeq.sciml.ai/stable/basics/common_solver_opts/#Miscellaneous)
 """
 ode_unstable_check(dt, u, semi, t) = isnan(dt)
 
 
 end # module
 
-using .TrixiMPIArrays: TrixiMPIArrays, TrixiMPIArray, ode_norm, ode_unstable_check
+@reexport using .TrixiMPIArrays: TrixiMPIArrays, TrixiMPIArray, ode_norm, ode_unstable_check
```

Comment:
This is the first time we export something from Trixi that is not exported from `src/Trixi.jl`, right? I would have preferred to have all exports in one place (since it makes it easier to figure out what's exported from Trixi and what isn't), but if you want to keep it here to improve spatial locality, I'm also fine with it.

Reply:
We also `@reexport` other stuff from Trixi. Since this file is only a temporary solution and everything is marked as experimental, I think it's fine for now. We will revisit this issue after some tome working with this stuff.

Reply:
👍